cmd/expiration-mailer/main.go

```diff
@@ -407,7 +407,7 @@ func (m *mailer) findExpiringCertificates(ctx context.Context) error {
 
 		var certs []certDERWithRegID
 		var err error
-		if features.Enabled(features.ExpirationMailerUsesJoin) {
+		if !features.Enabled(features.ExpirationMailerUsesJoin) {
```

Comment:
Ooh good catch. Perhaps an argument for `getCerts2` to be renamed `getCertsUsingJoin` or something more descriptive.